docs/openapi.yaml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.0
 info:
   title: IUDX-AAA-Server
-  version: 3.5.0
+  version: '4.0'
```

Comment:
can it be changed to ``4.0.0`` similar to this -https://github.com/datakaveri/iudx-resource-server/blob/f9957047a34adb540b96e1c16c0eb4fad94f8703/docs/openapi.yaml#L17